cli/commands/update.go

```diff
@@ -97,6 +98,7 @@ func parseUpdateResponse(responseBytes []byte) (string, error) {
 }
 
 func doUpdate(optionsFile, packageVersion string) {
+	config.Command = "update"
```

Comment:
@benclarkwood yup - it's in memory. I'll move this to the actual handler method instead so it's only ever set upon invocations of that command.